tests/ui_tests/test_cluster_list_page.py

```diff
@@ -638,3 +641,86 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterStatusPage:
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_open_by_tab_cluster_status_page(self, app_fs):
+        cluster_main_page = ClusterMainPage(app_fs.driver, app_fs.adcm.url, 1).open()
+        cluster_main_page.open_status_tab()
+        cluster_status_page = ClusterStatusPage(app_fs.driver, app_fs.adcm.url, 1)
+        cluster_status_page.wait_page_is_opened()
+        cluster_status_page.check_all_elements()
+
+    def test_filter_cluster_status_page(
+        self, app_fs, create_community_cluster_with_host_and_service
+    ):
+        cluster, host = create_community_cluster_with_host_and_service
+        cluster.hostcomponent_set(
+            (host, cluster.service(name=SERVICE_NAME).component(name=COMPONENT_NAME))
+        )
+        cluster_status_page = ClusterStatusPage(app_fs.driver, app_fs.adcm.url, 1).open()
+        group_row = cluster_status_page.get_all_config_groups()[0]
+        cluster_status_page.click_collapse_all_btn()
+        cluster_status_page.wait_group_opened(group_row)
+        cluster_status_page.click_collapse_all_btn()
+        cluster_status_page.wait_group_closed(group_row)
+        cluster_status_page.config.click_on_group(title=SERVICE_NAME)
+        with allure.step("Check group rows"):
+            group_info = cluster_status_page.get_config_group_info(group_row)
```

Comment:
maybe you need to check that only one item was found?

Reply:
added